pipelines/internal/commands/run/run.go

```diff
@@ -476,6 +476,10 @@ func parse(line string) (*genomics.Action, error) {
 		action.Entrypoint = "bash"
 	}
 
+	if actionTimeout, ok := options["timeout"]; ok {
```

Comment:
It would be nice to parse the duration as a [Go duration value](https://golang.org/pkg/time/#ParseDuration) so that users can say something like '1h' and we will convert it correctly to the API required format